File: baselib/src/main/java/com/bride/baselib/BaseFragment.java
package com.bride.baselib;

import android.os.Bundle;
import android.util.Log;

import androidx.fragment.app.Fragment;

/**
 * <p>Created by shixin on 2019/3/22.
 */
public class BaseFragment extends Fragment {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.i("onCreate", getClass().getSimpleName());
    }

    @Override
    public void onDestroy() {
        super.onDestroy();
        Log.i("onDestroy", getClass().getSimpleName());
    }
}

File: baselib/src/main/java/com/max/baselib/BaseFragment.kt
package com.max.baselib

import android.os.Bundle
import android.support.v4.app.Fragment
import android.util.Log

/**
 *
 * Created by shixin on 2018/9/4.
 */
open class BaseFragment : Fragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Log.i("create", " ${this.javaClass.simpleName}")
    }

    override fun onDestroy() {
        super.onDestroy()
        Log.i("destroy", " ${this.javaClass.simpleName}")
    }
}
